build.xml

```diff
@@ -42,7 +42,6 @@
   	</exec>
   </target>
 
-
```

Comment:
The ant script was created in-order to call all the npm scripts in a bundle instead of calling them one by one. 

In future, we can convert this into a central script which can be called by npm instead of ant.

Reply:
:

Reply:
https://www.npmjs.com/package/npm-run-all

Reply:
Will check on this, Thank you.